src/arviz_plots/plots/distplot.py

```diff
@@ -97,6 +99,7 @@ def plot_dist(
 
           * "kde" -> passed to :func:`~arviz_plots.visuals.line_xy`
           * "ecdf" -> passed to :func:`~arviz_plots.visuals.ecdf_line`
+          * "hist" -> passed to :func: `~WIP`
```

Comment:
update this and add the new visual function to https://github.com/arviz-devs/arviz-plots/blob/main/docs/source/api/visuals.rst